src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -122,7 +135,8 @@ public void periodic() {
             targetTunable.get(),
             ControlType.kPosition, 
             0,
-            wristFeedforward.calculate(Math.toRadians(positionMotor.getEncoder().getPosition()), 0.0)
+            movingUp ? wristFeedforwardUp.calculate(Math.toRadians(positionMotor.getEncoder().getPosition()), 0.0) 
```

Comment:
write a getter function to ge tthe correct position in radians and use to lower line length